### DISCUSSION TITLE
Is anyone else having some issues with following the build guide?

### DISCUSSION THREAD

**Discussion by Sky-LeL** [KadeArchive/Kade-Engine#1431] [Q&A]
Ok so I had an issue lime build windows and it gave me the error: 
Error: Source path "C:/HaxeToolkit/haxe/lib/extension-webm/git/ndll/Windows64/extension-webm.ndll" does not exist so I used ctrl + f to search the build guide for extension-webm. So I first ran haxelib git extension-webm https://github.com/KadeDev/extension-webm it updated what I already have. Then I tried lime rebuild extension-webm windows but it gave the error

Error: ExternalInterface.cpp
C:\HaxeToolkit\haxe\lib\hxcpp\4,2,1\include\hx\OS.h(59): fatal error C1083: Cannot open include file: 'stddef.h': No such file or directory
Is anyone else having this problem? Also can anyone help me fix it?

**Comment by ItsAssasin** [comment]
Install Windows SDK, it worked for me

**Reply by FChesco** [reply]
I will try it, cuz I have some issues too.
